RecoLocalMuon/GEMRecHit/python/me0RecHits_cfi.py

```diff
@@ -3,5 +3,5 @@
 me0RecHits = cms.EDProducer("ME0RecHitProducer",
     recAlgoConfig = cms.PSet(),
     recAlgo = cms.string('ME0RecHitStandardAlgo'),
-    me0DigiLabel = cms.InputTag("simMuonME0ReDigis"),
+    me0DigiLabel = cms.InputTag("simMuonME0PseudoReDigis"),
```

Comment:
@slava77 After this PR no more name changes are foreseen. The naming that is proposed is similar to the other muon digitizers `simMuonME0Digis` vs `simMuonRPCDigis` or `simMuonGEMDigis`,`simMuonCSCDigis` and `simMuonDTDigis` for realistic digis. `simMuonME0PadDigis` vs `simMuonGEMPadDigis` for pads. Pseudo digis now have the string `Pseudo` in them to distinguish them from real digis. Re-digitized pseudo digis for Muon TDR studies are called `simMuonME0PseudoReDigis`. This naming convention was agreed upon with other GEM/ME0 developers.